packages/cx-layout/src/components/Layout/tokens/cxLayout.ts

```diff
@@ -53,15 +47,27 @@ const Base = asLayoutToken({
   }
 });
 
+/*
+ * Tailwind's container is specifially not used due to its feature it set's max-width
+ * to min-width of breakpoint.  So instead rely on ContainerWrapper to margin percent
+ * to contain content until we get to xl and then constrain by max-width.
+ */
+const ConstrainSite = asLayoutToken({
+  Theme: {
+    HeaderWrapper: cxSpacing.WithSiteXLConstraint,
+    FooterWrapper: cxSpacing.WithSiteXLConstraint,
+  },
+});
+
 const Header = asLayoutToken({
   Components: {
-    SiteHeader: as(cxHeader.Default),
+    Header: as(cxHeader.Default),
```

Comment:
I don't understand the purpose of these `Header` and `Footer` tokens.  Why not simply include `cxHeader.Default` and `cxFooter.Default` in the `Base` token?